### PR TITLE
Migrate to GitHub

### PR DESCRIPTION
**Pull request by wyrzyk**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -21,16 +21,16 @@ Adding a requirement of a major version of a dependency is breaking a contract.
 Dropping a requirement of a major version of a dependency is a new contract.
 
 ## [Unreleased]
-[Unreleased]: https://bitbucket.org/atlassian/workspace/branches/compare/master%0Drelease-2.0.0
+[Unreleased]: https://github.com/atlassian/workspace/compare/release-2.0.0...master
 
 ## [2.0.0] - 2018-09-03
-[2.0.0]: https://bitbucket.org/atlassian/workspace/branches/compare/release-2.0.0%0Drelease-1.1.0
+[2.0.0]: https://github.com/atlassian/workspace/compare/release-1.1.0...release-2.0.0
```
Comment (dagguh) [inline CHANGELOG.md:27]: These diffs look pretty nice :)

**Review by dagguh — COMMENTED** [review]
Let's add Travis badge

**Review by dagguh — CHANGES_REQUESTED** [review]
I'd link back to Bitbucket to discover the past pull requests

File: build.gradle.kts [hunk 2]
```diff
@@ -2,7 +2,7 @@ val kotlinVersion = "1.2.30"
 
 plugins {
     kotlin("jvm").version("1.2.30")
-    id("com.atlassian.performance.tools.gradle-release").version("0.4.3")
+    id("com.atlassian.performance.tools.gradle-release").version("0.5.0")
```
Comment (dagguh) [inline build.gradle.kts:5]: This caused the build to fail:
```
org.gradle.api.publish.PublicationContainer.register(Ljava/lang/String;Ljava/lang/Class;Lorg/gradle/api/Action;)Lorg/gradle/api/NamedDomainObjectProvider;
```
This API is available in Gradle 5.0
I had to do it in https://github.com/atlassian/jira-actions/pull/2/commits/6c004ba678b076509bc52e7700e1b4aa2a88618a

**Review by dagguh — CHANGES_REQUESTED** [review]

**Review by sebapawlak — DISMISSED** [review]

**Comment by wyrzyk** [comment]
> I'd link back to Bitbucket to discover the past pull requests

I see no valuable information in the past PRs in the workspace repository. https://bitbucket.org/atlassian/workspace/pull-requests/?state=MERGED

**Review by dagguh — APPROVED** [review]

**Pull request merged**